src/ctia/http/routes/common.clj

```diff
@@ -99,6 +108,28 @@
     {:gte from
      :lt to-or-now}))
 
+(s/defn prep-es-fields-schema :- (s/protocol s/Schema)
+  "Conjoins ES :fields onto search-parameters."
+  [search-query-params :- (s/maybe (s/protocol s/Schema))]
+  (let [fields-schema  (st/get-in search-query-params [:fields])
```

Comment:
We need to review the `fields` value in each entity to ensure that they really cover the fields we want to search in.

Reply:
What do you mean? Just a manual audit or something? That's easy, we can open Swagger and it will show all the fields for every entity where the search will be performed (by default)

Reply:
> Just a manual audit or something? 

Yes somehow. `fields` was not particularly leveraged to retrieve partial documents on search. So it's very likely that some interesting fields to search in are not available.